src/pages/blog/starlingx-deploying-your-app-3.md

```diff
@@ -0,0 +1,250 @@
+
+Please, pay attention to correctly configure the files in `./output/poc-starlingx-app/plugins/k8sapp_poc_starlingx_app`:
+* **helm/poc_starlingx.py**: this file is responsible for integrating the application with the StarlingX system.
+* **kustomize/kustomize_poc_starlingx_app.py**: this file is responsible for updating top-level kustomization resource list.
+* **lifecycle/lifecycle_poc_starlingx_app.py**: this file configures how StarlingX will perform the lifecycle actions for the application opperator.
+
+After the plugin files are configured, I'll run the app-gen-tool command again, but now with the --package-only parameter.
+
+```sh
+(venv)~$: stx-app-generator -i app_manifest.yaml -o ./output --package-only
+
+...
+Plugin wheels generated!
+Checksum generated!
+FluxCD App tarball generated at /home/tomas-barros/blog-post-3/output/poc-starlingx-app/poc-starlingx-app-1.0.0.tgz
+```
+
+Alright! We can see the `poc-starlingx-app-1.0.0.tgz`. Now we only have to send this .tgz to a StarlingX cluster to install it.
+
+## Installing the App
+
+After copying your app package to your StarlingX cluster(in my case a virtual AIO-SX, we can now proceed to install it. But before installing it, please guarantee that your platform-integ-apps application is installed.
+
+```
+source /etc/platform/openrc
+system application-show platform-integ-apps | grep applied 
+```
+if the following output is:
+```
+| status        | applied                              |
+```
+proceed. Otherwise, you must need to wait unti it finishes applying.
+
+```
+sysadmin@controller-0:~$ ls
+admin-login.yaml  dashboard-values.yaml  poc-starlingx-app-1.0.0.tgz
+ansible.log       localhost.yml          token.txt
+
+```
+
+```
+sysadmin@controller-0:~$ system application-upload poc-starlingx-app-1.0.0.tgz
+```
+now, check that application was successfully uploaded:
+```
+system application-show poc-starlingx-app | grep uploaded
+```
+if the output is:
+```
+| status        | uploaded                           |
+```
+then it was successfully uploaded. Otherwise, wait for it to finish uploading or wait check common troubleshooting commands.
+
+Now, let's apply the app:
+```
+[sysadmin@controller-0 ~(keystone_admin)]$ system application-apply poc-starlingx-app
+``` 
+
+If you never installed a StarlingX app before, know that it is normal to take some minutes before finishing the applying command. We can check that the application was successfuly applied:
+
+```
+[sysadmin@controller-0 ~(keystone_admin)]$ system application-show poc-starlingx-app | grep applied
+| status        | applied
+```
+
+If you followed me until here, and your application was successfully applied, then we're done! In case you run into any problems, please check the troubleshooting section.
+
+## Troubleshooting
```

Comment:
Should this section exist? Since troubleshooting can be so much more than just this, it seems to me that maybe we shouldn't add this.

Reply:
I think this section can be useful for possible inexperienced users reading the blog post. But i'll add a note that troubleshooting can be more complicated than this.